FantasmoSDK/Classes/Network/FMApiRouter.swift

```diff
@@ -11,6 +11,7 @@ struct FMApiRouter {
     enum ApiEndpoint: String {
         case localize = "image.localize"
         case zoneInRadius = "parking.in.radius"
+        case deviceCharacteristics
```

Comment:
This is unrelated to the task at hand. And there is no spec suggesting that we need another endpoint.

Reply:
With this I must agree